queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth between '2016-01-01' and '2019-12-31';
+SELECT name FROM animals WHERE neutered = true and escape_attempts<3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' or name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true; 
+SELECT * FROM animals WHERE NOT name = 'Gabumon';
```

Comment:
I believe what you're trying to say here is `WHERE name NOT  = gabumon` not the other way 💡

Reply:
Thanks. I tried what you suggested but there is a syntax error. 

I am using the syntax from [this](https://www.w3schools.com/sql/sql_and_or.asp) page and is working properly.

![image](https://user-images.githubusercontent.com/38335710/135125799-4a815c88-4431-446b-9954-ce2a72c12776.png)